backend/Dockerfile.cacheserver

```diff
@@ -31,7 +31,11 @@ RUN go-licenses csv ./backend/src/cache > /tmp/licenses.csv && \
     diff /tmp/licenses.csv backend/third_party_licenses/cache_server.csv && \
     go-licenses save ./backend/src/cache --save_path /tmp/NOTICES
 
-FROM alpine:3.8
+FROM alpine:3
+
+RUN adduser -S appuser
```

Comment:
For tracking purposes, could you please add a comment to the PR's description on why do we need to use the user account?

Reply:
done